zerver/views/auth.py

```diff
@@ -251,7 +257,7 @@ def login_or_register_remote_user(request: HttpRequest, email: str,
     """
     if user_profile is None or user_profile.is_mirror_dummy:
         return register_remote_user(request, email, full_name,
-                                    is_signup=is_signup,
+                                    user_avatar_url, is_signup=is_signup,
```

Comment:
Most of the changes in this file will become unnecessary when we merged #14015, which I'm guessing will be before this merges.